rpmfusion-free-rawhide.repo

```diff
@@ -3,7 +3,8 @@ name=RPM Fusion for Fedora Rawhide - Free
 #baseurl=http://download1.rpmfusion.org/free/fedora/development/$basearch/os/
 metalink=https://mirrors.rpmfusion.org/metalink?repo=free-fedora-rawhide&arch=$basearch
 enabled=1
-gpgcheck=0
+gpgcheck=1
```

Comment:
`gpgcheck` on rawhide: Understood. If one cannot provide signed packages, he should rather not provide any packages. This is open gates for any malware or software criminals.

`repo_gpgcheck` works fine on your repositories. After enabling this option on your repos, I have to manually confirm the GPG key and then the repo works as before. Afaik mirrormanager solves this job for you.